### PR TITLE
Add "Withdraw stake" and "Claim withdraw" modal windows

### PR DESCRIPTION
**Pull request by goodsoft**
If there is available ordered withdraw for claiming, we generate both withdraw and claim modal windows, and present user a question modal.

### CODE REVIEW

File: apps/block_scout_web/assets/js/pages/stakes.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { openBecomeCandidateModal } from './stakes/become_candidate'
 import { openRemovePoolModal } from './stakes/remove_pool'
 import { openMakeStakeModal } from './stakes/make_stake'
 import { openMoveStakeModal } from './stakes/move_stake'
+import { openWithdrawStakeModal } from './stakes/withdraw_stake'
```
Comment (Heimdell) [inline apps/block_scout_web/assets/js/pages/stakes.js]: Two import above are duplicates.

File: apps/block_scout_web/assets/js/pages/stakes.js [hunk 2]
```diff
@@ -86,6 +87,7 @@ if ($stakesPage.length) {
     .on('click', '.js-remove-pool', () => openRemovePoolModal(store))
     .on('click', '.js-make-stake', event => openMakeStakeModal(event, store))
     .on('click', '.js-move-stake', event => openMoveStakeModal(event, store))
+    .on('click', '.js-withdraw-stake', event => openWithdrawStakeModal(event, store))
```
Comment (Heimdell) [inline apps/block_scout_web/assets/js/pages/stakes.js]: Yet another dups above.

File: apps/block_scout_web/assets/js/pages/stakes/withdraw_stake.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  const $modal = $(msg.html)
+  setupChart($modal.find('.js-stakes-progress'), msg.self_staked_amount, msg.staked_amount)
+  $modal.find('.btn-full-primary.withdraw').click(() => {
+    withdrawStake($modal, address, store)
+    return false
+  })
+  $modal.find('.btn-full-primary.order-withdraw').click(() => {
+    orderWithdraw($modal, address, store)
+    return false
+  })
```
Comment (saneery) [inline apps/block_scout_web/assets/js/pages/stakes/withdraw_stake.js]: As I remember these buttons `Withdraw now` and `Ordered withdrawal` should be active and inactive depending on [maxWithdrawAllowed](https://poanetwork.github.io/posdao-contracts/docs/abstracts_StakingBase.html#maxWithdrawAllowed) and [maxWithdrawOrderAllowed](https://poanetwork.github.io/posdao-contracts/docs/abstracts_StakingBase.html#maxWithdrawOrderAllowed)

Comment (goodsoft) [reply apps/block_scout_web/assets/js/pages/stakes/withdraw_stake.js]: They are (de)activated in HTML template

**Pull request merged**